### DISCUSSION TITLE
Bulk delete Accounts that never logged in and Send email to everybody

### DISCUSSION THREAD

**Discussion by Y0ngg4n** [dani-garcia/vaultwarden#4530] [Ideas]
Hi as my instance is getting bigger and bigger it would be nice to have a bulk delete button to delete all accounts that never logged in are not verified and have no entries. 
Also it would be nice to have the option to send emails out to every user.

**Comment by Y0ngg4n** [comment]
There are so many users on my instance that the admin page wont load the users page. This is really needed!

**Comment by BlackDex** [comment]
You could do this by running calls to the Vaultwarden Admin Backup API your self of course.

As we do not (yet) have true paging support in our admin environment, i would guess loading could take some time, but in the end should still load of course.

**Comment by BlackDex** [comment]
Also, well written PR's are always welcome :).

**Comment by Y0ngg4n** [comment]
@BlackDex i wrote a python script to call the admin api to delete old users. Will link it here soon. So we can close it.

**Comment by Y0ngg4n** [comment]
The script: https://gist.github.com/Y0ngg4n/99a7755484393c2fe5e9f8c958d97ac0
